setup.py

```diff
@@ -187,7 +186,8 @@ def run(self):
 
       scripts=SCRIPTS,
       include_package_data=True,
-      package_data={'static': ['static/*']},
+      package_data={'static': ['static/*'],
+                    'control_node_serial_interface':'control_node_serial/control_node_serial_interface'},
```

Comment:
this is a pep8: line exceeds 80 characters (I'm wondering why the CI still pass

Reply:
code_check is not run in CI, that's why :-)